src/Validation/Validation.php

```diff
@@ -1559,6 +1559,40 @@ public static function hexColor($check)
         return static::_check($check, '/^#[0-9a-f]{6}$/iD');
     }
 
+    /**
+     * Check that the input value is a valid IBAN number
+     *
+     * @param string $check The value to check
+     * @return bool Success
+     */
+    public static function iban($check)
```

Comment:
https://www.iban.com/structure.html

This validation method will pass IBANs that have wrong country codes or country code missmatches.
I am not saying that this should be fixed but the doc header should quote what the method does and what not.
I'd also say it should be `ibanSyntax($value)`.

The method checks for the country code to exist, the checksum to exist and then checks the body with Luhn algorithm

Reply:
iban() should suffice, the syntax sounds more like spacing check (or length only) or sth.

Reply:
it does nothing more than that. it does not validate an iban(). If it were it would be required to check the country code against the length, and possibly even the national bank codes against a database https://en.wikipedia.org/wiki/International_Bank_Account_Number#IBAN_formats_by_country

It is a pure stateless syntax check with. Validating an IBAN *for real* requires much more work. Thus `ibanSyntax()` hints you very clearly that you are only checking the form (uppercase, no whitespaces, max length, country code exists at right spot, checksum exists at right spot, body matches against checksum via Luhn)

Reply:
What about creditCard, ip ones, url etc?
They all also only do Syntax check, that is already very obvious here from the scope.